product.json

```diff
@@ -29,9 +29,8 @@
 	"documentationUrl": "https://go.microsoft.com/fwlink/?linkid=862277",
 	"commit": "9ca6200018fc206d67a47229f991901a8a453781",
 	"date": "2017-12-15T12:00:00.000Z",
+	"recommendedExtensions": [],
 	"extensionsGallery": {
-		"serviceUrl":"https://raw.githubusercontent.com/Microsoft/sqlopsstudio/release/extensions/extensionsGallery.json"
-	},
-	"recommendedExtensions": [
-	]
+		"serviceUrl": "https://raw.githubusercontent.com/Microsoft/sqlopsstudio/release/extensions/extensionsGallery.json"
```

Comment:
Did you still want this in here by default?

Reply:
yes, I think it's useful to have it for dev builds 

---
In reply to: [174282072](https://github.com/Microsoft/sqlopsstudio/pull/902#discussion_r174282072) [](ancestors = 174282072)